### PR TITLE
MNT Use React Testing Library

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1503

### CODE REVIEW

File: client/src/components/LoginSession/tests/LoginSession-test.js [hunk 1]
```diff
@@ -1,69 +1,80 @@
...
+
+test('LoginSession should be hidden when complete', () => {
+  const { container } = render(
+    <LoginSession {...makeProps({
+      submitting: false,
+      complete: true
+    })}
+    />
+  );
+  expect(container.querySelector('.login-session.hidden')).not.toBeNull();
```
Comment (sabina-talipova) [inline client/src/components/LoginSession/tests/LoginSession-test.js]: Before we tested that we have only one element. Should we check quantity of elements?

Comment (sabina-talipova) [inline client/src/components/LoginSession/tests/LoginSession-test.js:69]: Could we check that we have only one element `'.login-session.hidden'`?

Comment (emteknetnz) [reply client/src/components/LoginSession/tests/LoginSession-test.js]: Change to test length

**Review by sabina-talipova — APPROVED** [review]
LGTM

**Pull request merged**